modules/oadp-backing-up-dpa-configuration-1-4-0.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="oadp-backing-up-dpa-configuration-1-4-0_{context}"]
+= Backing up the DPA configuration
```

Comment:
In the preview, this header is H3 while the empty header in the release notes file (Upgrading steps) is also H3. If this procedure is supposed to be nested, adjust the headers accordingly.

Reply:
This is same for the "Upgrading the OADP Operator" section and even in the previous release notes structure. 
Let me know if you have any suggestion for this

Reply:
fixed